data/meditation.py

```diff
@@ -0,0 +1,61 @@
+import datetime
```

Comment:
We already have this model defined here: https://github.com/ssw555tm012021Fall/halcyon-api/blob/main/data/room.py 
Kindly use that. Also, we don't need `encode_auth_token()` and `decode_auth_token()` functions in any files other than `data/employee.py`